src/tests/Regressions/coreclr/GitHub_76531/tailcallinvoker.il

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+.assembly extern legacy library mscorlib {}
+.assembly extern dependencytodelete {}
+.assembly 'lowlevel' {}
```

Comment:
```suggestion
.assembly 'tailcallinvoker' {}
```
Should the assembly name match the file name?

Reply:
Thank you for spotting this, I've forgotten to rename it after renaming the file itself locally.